Stripe/STPAnalyticsClient.m

```diff
@@ -252,7 +252,7 @@ - (void)logSetupIntentConfirmationAttemptWithConfiguration:(STPPaymentConfigurat
     NSMutableDictionary *payload = [self.class commonPayload];
     [payload addEntriesFromDictionary:@{
                                         @"event": @"stripeios.setup_intent_confirmation",
-                                        @"payment_method_type": paymentMethodType ?: @"unknown",
+                                        @"source_type": paymentMethodType ?: @"unknown",
```

Comment:
deliberately overloading `source_type` here like android does.

Reply:
We could also pass this as `payment_method_type` from the client and figure out what to do with it on the backend. For this PR maybe simpler to go with what you have.